docs/_sidebar.md

```
@@ -27,6 +27,7 @@
    - [State Management](development/state-management.md)
    - [Type System](development/type-system.md)
    - [UI Components](development/ui-components.md)
+   - [Validation](development/validation.md)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 3

(MD007, ul-indent)